locales/da.lua

```diff
@@ -0,0 +1,195 @@
+        impound_veh = '[E] - Beslaglæg køretøj',
+        store_veh = '[E] - Gem køretøj',
+        armory = 'Våbenlager',
+        enter_armory = '[E] Våbenlager',
+        finger_scan = 'Fingeraftryksscanning',
+        scan_fingerprint = '[E] Scan fingeraftryk',
+        trash = 'Papirkurv',
+        trash_enter = '[E] Papirkurv',
+        stash_enter = '[E] Gå ind i skabet',
+        target_location = 'Placeringen af ​​%{firstname} %{lastname} er markeret på dit kort',
+        anklet_location = 'Anklet placering',
+        new_call = 'Nyt opkald',
+        officer_down = 'Betjent %{lastname} | %{callsign} ned',
+        fine_issued = 'Bøde er blevet sendt til gerningsmanden',
+        receive_fine = 'Statens inddrivelse af gæld har automatisk inddrevet de skyldige bøder...'
+    },
+    evidence = {
+        red_hands = 'Røde hænder',
+        wide_pupils = 'Brede elever',
+        red_eyes = 'Røde øjne',
+        weed_smell = 'Lufter af ukrudt',
+        gunpowder = 'krudt på tøj',
+        chemicals = 'lugter kemikalie',
+        heavy_breathing = 'Trækker vejret tungt',
+        sweat = 'sveder meget',
+        handbleed = 'Blod på hænder',
+        confused = 'Forvirret',
+        alcohol = 'Lufter af alkohol',
+        heavy_alcohol = 'Lufter meget af alkohol',
+        agitated = 'Ophidset - tegn på meth-brug',
+        serial_not_visible = 'Serienummer er ikke synligt...',
+    },
+    menu = {
+        garage_title = 'Politibiler',
+        close = '⬅ Luk menu',
+        impound = 'Beslaglagte køretøjer',
+        pol_impound = 'Politiet beslaglæggelse',
+        pol_garage = 'Politigarage',
+        pol_armory = 'Politiets våbenlager',
+    },
+    email = {
+        sender = 'Den Centrale Retlige Inddrivelsesmyndighed',
+        subject = 'Gældsinddrivelse',
+        message = 'Kære %{value}. %{value2}, <br /><br />Central Judicial Collection Agency (CJCA) opkrævede de bøder, du modtog fra politiet.<br />Der er <strong>%{value3} kr.</strong> trukket fra din konto.<br /><br />Med venlig hilsen<br />Hr. I.K. Graai',
+    },
+    commands = {
+        place_spike = 'Placer Spike Strip (kun politiet)',
+        license_grant = 'Giv en licens til nogen',
+        license_revoke = 'Tilbagekald en licens fra nogen',
+        place_object = 'Placer/slet et objekt (kun politiet)',
+        cuff_player = 'Sæt håndjern på spiller (kun politiet)',
+        escort = 'Eskortespiller',
+        kaldesignal = 'Giv dig selv et kaldesignal',
+        clear_casign = 'Ryd område for hylstre (kun politiet)',
+        jail_player = 'Fængselsspiller (kun politiet)',
+        unjail_player = 'Fjern spilleren (kun politiet)',
+        clearblood = 'Ryd området for blod (kun politiet)',
+        seizecash = 'Beslaglæg kontanter (kun politiet)',
+        softcuff = 'Sæt Bløde Håndjern på spiller (kun politiet)',
+        camera = 'Se sikkerhedskamera (kun politiet)',
+        flagplade = 'Flag en plade (kun politiet)',
```

Comment:
flagplade should be flagplate

Reply:
its danish so no

Reply:
> its danish so no

The translation may be in danish but the word is :  flagplate = ...... , so yes